alas_but_one.py

```diff
@@ -35,13 +71,16 @@
                 print(e)
 
 ctr = Counter(word_list)
-if os.path.isfile(EXCLUSION_FILE):
-    exclusion_list = open(EXCLUSION_FILE).read()
-    for w,v in sorted(ctr.items(), key=lambda pair: pair[1], reverse=True):
-        if (v <= MAX_OCCUR):
-            if (not((w + " ") in exclusion_list)):
-                print("%s [%d]" % (w,v), file=open(OUTPUT_FILE, 'a'))
-else:
-    for w,v in sorted(ctr.items(), key=lambda pair: pair[1], reverse=True):
-        if (v <= MAX_OCCUR):
-            print("%s [%d]" % (w,v), file=open(OUTPUT_FILE, 'a'))
+for w,v in sorted(ctr.items(), key=lambda pair: pair[1], reverse=True):
+    addWord=((v <= MAX_OCCUR) and
+             (not(doExclusion) or
+                (doExclusion and not(w in exclusion_list))))
```

Comment:
Issue:
While this is one way to accomplish the task, longer boolean expressions are often error-prone in both interpretation and implementation, especially when a contributor wants to add new logic. The original variables were also named for speed rather than readability.
This boolean is also only referenced once, so it doesn't seem necessary to set a variable for it in the loop.

in a previous review comment, I mentioned [setting a boolean](https://github.com/ccho-mongodb/miscellaneous-tools/pull/5#discussion_r1184435869). This is still a potential solution, but I did not mean to imply that it should be added into this loop. The purpose was to indicate that the exclusion feature was enabled and contained words.

Suggestion:

The exclusion list prevents any of the terms from being considered for presentation, so they can be discarded earlier, such as before the call to ``sorted``.

I would recommend the solution in the [previous review comment](https://github.com/ccho-mongodb/miscellaneous-tools/pull/5#pullrequestreview-1412071000). To accomplish this, you can use the ``del ctr[<word from exclusion list>]`` call (see the [Counter objects documentation](https://docs.python.org/3/library/collections.html#counter-objects)). 

This logic to modify the Counter object could be placed in a separate function (for testability and maintainability) and be called when the exclusion list is enabled. This solution would offer an O(n) complexity rather than the current O(n*m) one.